src/components/masthead.css

```diff
@@ -56,6 +56,7 @@
       width: 50%;
       white-space: nowrap;
       overflow-x: auto;
+      overflow-y: scroll;
```

Comment:
Why scroll and not auto? Anyway should never be necessary to scroll horizontaly in the sidebar.